### PR TITLE
Bug 1263840 - [TV][Home] Move card description text to outside of but…

### PR DESCRIPTION
**Pull request by rexboy7**
…tons

### CODE REVIEW

File: tv_apps/smart-home/js/card_picker.js [hunk 1]
```diff
@@ -167,39 +169,43 @@
...
-      function createButtonHelper(card, parentType) {
+      function createCardElemHelper(card, parentType) {
         if(card instanceof Folder) {
           return;
         }
-        var appButton = CardUtil.createCardButton(card, true);
-        return appButton;
+        var nodeElem = document.createElement('div');
+        nodeElem.appendChild(CardUtil.createCardFragment(card, true));
+        that.gridView.appendChild(nodeElem);
```
Comment (luke-chang) [inline tv_apps/smart-home/js/card_picker.js]: Guess we shouldn't `appendChild` here, or those need to be sorted will be appended twice.

File: tv_apps/smart-home/js/card_util.js [hunk 2]
```diff
@@ -187,19 +188,16 @@
...
 
-    updateCardName: function(cardButton, card) {
-      var spans =
-           SharedUtils.nodeListToArray(cardButton.getElementsByTagName('span'));
-      spans.forEach(span => {
-        if (span.classList.contains('name')) {
-          this._localizeCardName(span, card);
-        }
-      });
+    updateCardName: function(nameElement, card) {
```
Comment (luke-chang) [inline tv_apps/smart-home/js/card_util.js]: According to the spec, we may need to update the second name span in the folder card, so I suggest you pass the entire card element as the first argument.

File: tv_apps/smart-home/js/home.js [hunk 3]
```diff
@@ -386,8 +386,8 @@
     },
 
     onCardUpdated: function(scrollable, card, idx) {
-      var cardButton = scrollable.getItemFromNode(scrollable.getNode(idx));
-      CardUtil.updateCardName(cardButton, card);
+      CardUtil.updateCardName(
+                          scrollable.getNode(idx).querySelector('name'), card);
```
Comment (luke-chang) [inline tv_apps/smart-home/js/home.js]: As mentioned above, I suggest you use `scrollable.getNode(idx)` here.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8058bc90df907c98ee88d80962680d8e6e7ae84e

**Pull request merged**